packages/client/src/v2-events/hooks/utils/PDFUtils.ts

```diff
@@ -0,0 +1,309 @@
+        )
+        .join(', ')
+    } else if (
+      typeof templateData[key] === 'object' &&
+      templateData[key] !== null
+    ) {
+      templateData[key] = formatAllNonStringValues(
+        templateData[key] as Record<string, TemplateDataType>,
+        intl
+      )
+    }
+  }
+  return templateData as Record<string, string>
+}
+
+const cache = createIntlCache()
+
+export function compileSvg(
+  templateString: string,
+  data: Record<string, any> = {},
+  state: IStoreState
+): string {
+  const intl = createIntl(
+    {
+      locale: state.i18n.language,
+      messages: state.i18n.messages
+    },
+    cache
+  )
+
+  const customHelpers = getHandlebarHelpers()
+
+  for (const helperName of Object.keys(customHelpers)) {
+    /*
+     * Note for anyone adding new context variables to handlebar helpers:
+     * Everything you expose to country config's here will become API surface area,
+     * This means that countries will become dependant on it and it will be hard to remove or rename later on.
+     * If you need to expose the full record, please consider only exposing the specific values you know are needed.
+     * Otherwise what happens is that we lose the ability to refactor and remove things later on.
+     */
+    const helper = customHelpers[helperName]({ intl })
+    Handlebars.registerHelper(helperName, helper)
+  }
+
+  Handlebars.registerHelper(
+    'intl',
+    function (this: any, ...args: [...string[], Handlebars.HelperOptions]) {
+      // If even one of the parts is undefined, then return empty string
+      const idParts = args.slice(0, -1)
+      if (idParts.some((part) => part === undefined)) {
+        return ''
+      }
+
+      const id = idParts.join('.')
+
+      return intl.formatMessage({
+        id,
+        defaultMessage: 'Missing translation for ' + id
+      })
+    } as any /* This is here because Handlebars typing is insufficient and we can make the function type stricter */
+  )
+
+  Handlebars.registerHelper(
+    'ifCond',
+    function (
+      this: any,
+      v1: string,
+      operator: string,
+      v2: string,
+      options: Handlebars.HelperOptions
+    ) {
+      switch (operator) {
+        case '===':
+          return v1 === v2 ? options.fn(this) : options.inverse(this)
+        case '!==':
+          return v1 !== v2 ? options.fn(this) : options.inverse(this)
+        case '<':
+          return v1 < v2 ? options.fn(this) : options.inverse(this)
+        case '<=':
+          return v1 <= v2 ? options.fn(this) : options.inverse(this)
+        case '>':
+          return v1 > v2 ? options.fn(this) : options.inverse(this)
+        case '>=':
+          return v1 >= v2 ? options.fn(this) : options.inverse(this)
+        case '&&':
+          return v1 && v2 ? options.fn(this) : options.inverse(this)
+        case '||':
+          return v1 || v2 ? options.fn(this) : options.inverse(this)
+        default:
+          return options.inverse(this)
+      }
+    }
+  )
```

Comment:
Let's drop out the handlebar helpers that can be implemented in country config, so `ifCond` and `formatDate`. And overall, let's aim to do the minimum possible implementation with the least amount of code. We can always add more in later